### ISSUE TITLE
Word Cloud Creation

### ISSUE THREAD

**Issue by rexdivakar** [Open-Dev-Community/Tweet_data_hunter#5]
Creation of wordcloud or plot the distinct features extracted from the data loader.

**Comment by hpnightowl** [comment]
can i take this issue i can create a word cloud from it

**Comment by rexdivakar** [comment]
Issue assigned to @hpnightowl 
🌟 Star it 🍴Fork it 🤝 Contribute to it!
Happy Coding

**Comment by hpnightowl** [comment]
so do i have to make a script for creation of word cloud? or the image of word cloud is needed in readme

**Comment by rexdivakar** [comment]
Script for creating of word cloud @hpnightowl

**Comment by hpnightowl** [comment]
i have made a script like this via flask is this good to go?


from wordcloud import WordCloud, STOPWORDS
from flask import Flask, request
from flask_restful import Resource, Api

app = Flask(__name__)
api = Api(app)


class wordCloudservice(Resource):
    def post(self):
        ip_json = request.get_json()
        # get file name and result filename
        wordFile = ip_json["Filename"]
        picFile = ip_json["ResultFilename"]
        dataset = open(wordFile, "r").read()
        # utility function to create wordcloud

        def create_word_cloud(string):
            cloud = WordCloud(
                background_color="white", max_words=200,
                stopwords=set(STOPWORDS)
            )
            cloud.generate(string)
            cloud.to_file(picFile)

        dataset = dataset.lower()
        create_word_cloud(dataset)
        # change path to see word cloud generated
        return "Wordcloud can be found at " + picFile, 200


# endpoint
api.add_resource(wordCloudservice, "/wordCloudService")

if __name__ == "__main__":
    app.run(debug=True)

**Comment by rexdivakar** [comment]
yeah this looks great, create it as a separate class file and raise a PR for approval @hpnightowl

**Comment by hpnightowl** [comment]
ok 👍
